packages/pentaho/src/Canvas/BottomPanel/BottomPanel.stories.tsx

```diff
@@ -33,3 +37,138 @@ export const Main: StoryObj<HvCanvasBottomPanelProps> = {
     return <MainStory {...args} />;
   },
 };
+
+export const Test: StoryObj = {
```

Comment:
We need to update the bottom panels tabs to use this story now: https://github.com/lumada-design/hv-uikit-react/blob/master/packages/pentaho/src/Canvas/BottomPanel/BottomPanel.spec.tsx#L12

Reply:
yes, I noticed 🙈